package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.3",
   "description": "Javascript module template.",
   "main": "index.js",
-  "module": "dist/module/index.js",
+  "module": "dist/brainblocks.js",
```

Comment:
Can't consume that module because it's not ES5.

Reply:
I believe the `module` proposal is for compiled code, but still with `import` statements to allow for tree-shaking. Is your bundler not using the `index` setting?

Reply:
The `module` proposal is for ES6, but it will automatically be used by Webpack and Rollup and doesn't expect flow typings in those scenarios.  So I guess the ideal setup would be:

1. Update your `index.js` to point at the compiled version of your lib (still pointing at `./dist/xcomponent-demo`)
2. Either remove `module` or update it to use vanilla ES6 (no flow)

https://stackoverflow.com/a/47537198